It looks like `io.cdap.cdap.spark.SparkTest` tests fails:
```
2021-06-11 01:00:26,274 - INFO  [redirect output:i.c.c.c.l.RedirectedPrintStream@133] - Traceback (most recent call last):
2021-06-11 01:00:26,275 - INFO  [redirect output:i.c.c.c.l.RedirectedPrintStream@133] -   File "/tmp/cdap/cdap/cdap-unit-test/target/junit4236394413703048106/junit2908889939141274596/tmp/1623373223112-0/default.TestSparkApp.PythonSpark.64007c88-ca50-11eb-a13e-b2041ff2a642
.py", line 20, in <module>
2021-06-11 01:00:26,279 - INFO  [redirect output:i.c.c.c.l.RedirectedPrintStream@133] -     from cdap.pyspark import SparkExecutionContext
2021-06-11 01:00:26,280 - INFO  [redirect output:i.c.c.c.l.RedirectedPrintStream@133] -   File "/tmp/cdap/cdap/cdap-unit-test/target/junit4236394413703048106/junit2908889939141274596/tmp/1623373223112-0/cdap-pyspark-lib.zip/cdap/pyspark/__init__.py", line 18, in <module>
2021-06-11 01:00:26,281 - INFO  [redirect output:i.c.c.c.l.RedirectedPrintStream@133] -   File "/tmp/cdap/cdap/cdap-unit-test/target/junit4236394413703048106/junit2908889939141274596/tmp/1623373223112-0/cdap-pyspark-lib.zip/cdap/pyspark/context.py", line 18, in <module>
2021-06-11 01:00:26,281 - INFO  [redirect output:i.c.c.c.l.RedirectedPrintStream@133] - ImportError: No module named builtins
```

--- a/cdap-spark-python/src/main/resources/cdap/pyspark/context.py
+++ b/cdap-spark-python/src/main/resources/cdap/pyspark/context.py
@@ -15,8 +15,7 @@
 # the License.
 
 from __future__ import print_function
-from builtins import str
-from builtins import object
+
 import os
 from py4j.java_gateway import java_import, JavaGateway
 from threading import RLock

